pkg/controller/compliance/compliance_controller.go

```diff
@@ -134,6 +134,7 @@ func add(mgr manager.Manager, c controller.Controller) error {
 			render.ElasticsearchComplianceControllerUserSecret, render.ElasticsearchComplianceReporterUserSecret,
 			render.ElasticsearchComplianceSnapshotterUserSecret, render.ElasticsearchComplianceServerUserSecret,
 			render.ComplianceServerCertSecret, render.ManagerInternalTLSSecretName, certificatemanagement.CASecretName,
+			relasticsearch.PublicCertSecret,
```

Comment:
Looks like the compliance controller uses this secret in the bundle without setting up a watch for it. LGTM!